src/App.jsx

```diff
@@ -1,11 +1,14 @@
 import Navbar from "@components/Navbar";
 import TaskBoard from "@components/TaskBoard";
+import { useState } from "react";
 
 function App() {
+    const [isSearching, setIsSearching] = useState(false);
```

Comment:
Can't we use redux instead of state? Also `App` should be clean, there shouldn't be state in the `App` component